mailpoet/lib/WooCommerce/Helper.php

```diff
@@ -78,4 +81,21 @@ public function getRawPrice($price, array $args = []) {
   public function getAllowedCountries(): array {
     return (new \WC_Countries)->get_allowed_countries() ?? [];
   }
+
+  public function wasMailPoetInstalledViaWooCommerceOnboardingWizard(): bool {
+    $wp = ContainerWrapper::getInstance()->get(WPFunctions::class);
+    $installedViaWooCommerce = false;
+    $wooCommerceOnboardingProfile = $wp->getOption('woocommerce_onboarding_profile');
+
+    if (
+      !empty($wooCommerceOnboardingProfile)
```

Comment:
I might replace the `!empty` with `is_array`, just in case someone decided to come along and `update_option('woocommerce_onboarding_profile', new \stdClass())` or something equally as unlikely and ridiculous.

Reply:
Good point! I failed to consider this. I added a new commit making the change that you suggested.